Description: script tries to install the binaries in $GOBIN, which is
undesired. We install the binaries manually in the correct locatioin.
Upstream: No
diff --git a/scripts/build.sh b/scripts/build.sh
index 0dae7b0..7aa977b 100755
--- a/scripts/build.sh
+++ b/scripts/build.sh
@@ -44,16 +44,6 @@ ${GO_CMD} build \
     -tags "${BUILD_TAGS}" \
     .
 
-# Move all the compiled things to the $GOPATH/bin
-OLDIFS=$IFS
-IFS=: FIRST=($GOPATH) BIN_PATH=${GOBIN:-${FIRST}/bin}
-IFS=$OLDIFS
-
-# Ensure the go bin folder exists
-mkdir -p ${BIN_PATH}
-rm -f ${BIN_PATH}/bao
-cp bin/bao ${BIN_PATH}
-
 # Done!
 echo
 echo "==> Results:"
